Source/WebCore/dom/FullscreenManager.h

```diff
@@ -128,6 +128,22 @@ class FullscreenManager final : public CanMakeWeakPtr<FullscreenManager>, public
 #endif
 };
 
+class CompletionHandlerScope final {
```

Comment:
This could be a nested private class.  Either way, I think I should generalize CompletionHandlerCallingScope and get rid of it.